### ISSUE TITLE
Requirements aren't listed

### ISSUE THREAD

**Issue by datakid** [Oshlack/JAFFA#14]
JAFFA requires Groovy, but that's not listed anywhere - until after you run it and it fails with error message "could not find or load main class org.codehaus.groovy.tools.GroovyStarter".

**Comment by datakid** [comment]
Ok, I've solved this problem. 

The bpipe that is linked to in your install script (0.9.8.2_rc2) no longer exists afaict - it might be a minor outage, but there are more up to date bpipes anyway, and they have performance improvements (both in underlying groovy updates and in bpipe updates).

The error that caused this search was 

`Could not find the main class: org.codehaus.groovy.tools.GroovyStarter.`

which was a function of jaffa/tools/bpipe only having a bin dir - the lib was missing. bpipe hadn't installed correctly. I deleted it all and ran ./install_linux64 again and it failed on the URL. So I installed by hand with success.

I am currently naively running jaffa successfully using the current bpipe ([0.9.9.2](https://github.com/ssadedin/bpipe/releases/tag/0.9.9.2)). The only concern I have is that there is a small change to how bpipe works wrt to the transform function. Is this used by JAFFA? Regardless, it now seems to be functional.
